src/shared/config.ts

```diff
@@ -18,3 +19,5 @@ export const langOptions = [
 ];
 
 export const PORTAL_ID = 'header-dropdown-portal';
+
+export const applicationCurrency = ref('$');
```

Comment:
Preferrably to follow the existing style - define a composable in `ui/composables`

```ts
export function useApplicationCurrency(): Ref<string>
```